includes/wp-admin/class-settings.php

```diff
@@ -350,6 +350,9 @@ public static function settings_page() {
 				\add_thickbox();
 				\wp_enqueue_script( 'updates' );
 				break;
+			case 'settings':
+				\update_option( 'activitypub_launchpad_settings_visited', true );
```

Comment:
I wonder if this option is even needed. Could we use the `activitypub_actor_mode` option instead and see if it has a value? The task would be completed once they selected an actor mode and saved it.

Reply:
what if the user does not change it, because they are fine with the default?

Reply:
We could consider removing the default in the settings callback so the radio buttons are all empty. I guess it depends on whether we want them to understand it and make a conscious choice, or just consider it.

Reply:
But this would force the user to go through every step, no "work out of the box", no easy skip.

Reply:
Yeah, agreed.